### ISSUE TITLE
Have a way to save the current crafting choices to be able to easily come back to it

### ISSUE THREAD

**Issue by Guillaume-Docquier** [Guillaume-Docquier/guillaume-docquier.github.io#31]
If I set out to craft a two handed mace, but I'm unable to craft it right now, I will have to come back to Crowcraft and redo my crafting choices to get the materials and crafting steps.

This can get tedious for non dedicated crafters who will typically figure out what to craft, go harvest and then craft. This process can span multiple playing sessions.

This would also allow players to create a "wishlist" and send it to their guild crafters.

**Comment by recatek** [comment]
Would be nice to do this as a URL with parameters (similar to how crowcaine.wiki builds work) so it could integrate with other tools.

**Comment by Guillaume-Docquier** [comment]
Yep, I think url parameters is the most straightforward way to do this.
We'll probably want the url parameters to be populated as you work in Crowcraft, and the app should parse them on page load.
We will not want to share the crafting configuration via url, however. This will be saved per user in localstorage (see #40).

**Comment by Guillaume-Docquier** [comment]
Closing since Crowfall is dead
